src/pages/Pool/PositionPage.tsx

```diff
@@ -346,7 +346,32 @@ const useInverter = ({
   }
 }
 
-export function PositionPage() {
+const PositionPageUnsupportedContent = () => (
```

Comment:
Can you make this a function to be more idiomatic?

```suggestion
function PositionPageUnsupportedContent() {
```